src/features/cartographie/components/lieux-mediation-numerique-details/lieux-mediation-numerique-details.component.html

```diff
@@ -27,12 +27,13 @@
         [distance]="lieuMediationNumerique.distance"></app-informations-pratiques>
     </li>
     <li
-      class="list-group-item"
+      class="list-group-item border-bottom-0"
```

Comment:
```suggestion
      class="list-group-item"
```
Je ne vois pas cette bordure sur le proto

![image](https://github.com/anct-cartographie-nationale/client-application/assets/68475403/334a7d6d-18a1-4e2e-9e1e-0647b023e834)